ti/fitImage-am57xx.its

```diff
@@ -0,0 +1,63 @@
+/dts-v1/;
+
+/ {
+	description = "U-Boot fitImage";
+	#address-cells = <1>;
+
+	images {
+		kernel@1 {
+			description = "Linux kernel";
+			data = /incbin/("zImage.sec");
+			type = "kernel";
+			arch = "arm";
+			os = "linux";
+			compression = "none";
+			load = <0x82000000>;
+			entry = <0x82000000>;
+		};
+
+		am57xx-evm.dtb {
+			description = "Flattened Device Tree blob";
+			data = /incbin/("am57xx-evm.dtb.sec");
+			type = "flat_dt";
+			arch = "arm";
+			compression = "none";
+		};
+
+		am57xx-evm-reva3.dtb {
+			description = "Flattened Device Tree blob";
+			data = /incbin/("am57xx-evm-reva3.dtb.sec");
+			type = "flat_dt";
+			arch = "arm";
+			compression = "none";
+		};
+
+		optee {
+			description = "OPTEE OS Image";
+			data = /incbin/("tee.bin.sec");
+			type = "tee";
+			arch = "arm";
+			compression = "none";
+			load = <0xbdb000e4>;
```

Comment:
Probably, touching firewalled memory, even pre-fetches and speculative access, will throw exceptions, so trying to load a binary to one will certainly fail spectacularly.